mocha-client-tests/addon/background.js

```diff
@@ -0,0 +1,13 @@
+var Background = {
+    receiveMessage: function(msg, sender, sendResponse) {
+        if (msg && msg.action && Background.hasOwnProperty(msg.action)) {
+            return Background[msg.action](msg, sender, sendResponse);
+        }
```

Comment:
it would be nice if this had an else clause to warn about missed messages. It could help someone who may be editing this code. Something like:
````
... else {
    console.warning('No handler for message: ' + JSON.stringify(msg));
}
````